release/src/router/rom/webs_scripts/merlin_webs_update.sh

```diff
@@ -43,8 +43,7 @@ else
 	firmver=$(echo $fullver | cut -d. -f2)
 	buildno=$(echo $fullver | cut -d. -f3)
 
-	extendno=$(grep $model /tmp/wlan_update.txt | sed s/.*#EXT//;)
-	extendno=$(echo $extendno | sed s/#.*//;)
+	extendno=$(grep "$model" /tmp/wlan_update.txt | sed -n 's/.*#EXT\([0-9]*\).*/\1/p' | awk 'END {print}')
 	lextendno=$(echo $extendno | sed s/-g.*//;)
```

Comment:
This fix changes the value & intended meaning of the "**extendno**" variable, and therefore makes it incompatible with previously expected result & functionality.

**EXPECTED EXAMPLE RESULT**:
extendno=2-gnuton1

There are many syntax issues that could be improved on this script, but to maintain compatibility with previous functionality, the simplest changes to make it work with MelinAU are the following:

**CURRENT LINE 40**:
`fullver=$(grep $model /tmp/wlan_update.txt | sed s/.*#FW//;)`

**FIXED LINE 40**:
`fullver="$(grep "$model" /tmp/wlan_update.txt | tail -n1 | sed 's/.*#FW//')"`


**CURRENT LINE 46**:
`extendno=$(grep $model /tmp/wlan_update.txt | sed s/.*#EXT//;)`

**FIXED LINE 46**:
`extendno="$(grep "$model" /tmp/wlan_update.txt | tail -n1 | sed 's/.*#EXT//')"`

This is, of course, in addition to commenting out or changing current **line 2**:  ` set -x;`

My 2 cents.

Reply:
@Martinski4GitHub 

Good point, I was only focusing on fixing the value and not thinking about preserving the original intended functionality, this could be a problem as you mentioned. 

It should remain the "gnuton1"
I'll quickly adjust this so it maintains the complete extendo value!

As always your 2 cents on the full picture goes a long way!

Reply:
@Martinski4GitHub 

Done :) I confirmed the output now also remains the "gnuton1" to maintain compatibility with other functions he may need "extendo" for.

Expected output has gone from:

`echo ---- Stable available 3004.388.6_2----`

To:

`echo ---- Stable available 3004.388.6_2-gnuton1----`

As expected, but also resolves the issues with MerlinAU.

Reply:
I was in the MerlinAU Mindset what can I say... We usually only extract the version 😉